### PR TITLE
[templates/angular-xmcloud][templates/node-xmcloud-proxy] Angular and XM Cloud Proxy Readme updates

### PR DESCRIPTION
**Pull request by yavorsk**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [x] Changelog updated
- [ ] Upgrade guide entry added

## Description / Motivation

This PR includes Angular for XM Cloud and XM Cloud Node Proxy Readme updates, according to lates feedback from PSS.

## Testing Details

- [ ] Unit Test Added
- [ ] Manual Test/Other (Please elaborate)

## Types of changes

- [x] Chore
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
 ﻿# Angular for XMCloud
 
-> Sitecore JSS Angular App for XM Cloud. For the current release this feature is experimental.
+> Sitecore JSS Angular App for XM Cloud. For the current release this feature currently in public beta.
+> If you encounter any issues or bugs feel free to submit new issue in [Jss GitHub](https://github.com/Sitecore/jss)
 
 [Documentation](<TODO>)
 
-This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration. The supported key features are as follows:
+This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration. When used with the Node The supported key features are as follows:
```
Comment (stasmaxymov) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: typo, capital letter:  "When used with the Node **The**"

File: packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md [hunk 2]
```diff
@@ -72,9 +78,11 @@ The following environment variables can be set to configure the Proxy sample ins
 You should be able to see the following message:
 `server listening on port 3000!`.
 
-## Deploy to Netlify
+## Deployment options
+
+### Deploy to Netlify
 
-`NOTE: If you are using the Angular starter from the XM-Cloud Foundation repository within a monorepo, please skip to Step 3.`
+Follow the steps below to deploy your starter SPA (instructions are for Angular SPA, more to follow in the future). The Neylify deployment has been tested and can be used as editing host in XM Cloud.
```
Comment (stasmaxymov) [inline packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md]: typo: Neylify

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 3]
```diff
@@ -1,60 +1,89 @@
 ﻿# Angular for XMCloud
 
-> Sitecore JSS Angular App for XM Cloud. For the current release this feature is experimental.
+> Sitecore JSS Angular App for XM Cloud. For the current release this feature is in public beta.
+> If you encounter any issues or bugs, please submit a new issue in the [Jss GitHub Repo](https://github.com/Sitecore/jss)
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: JSS

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 4]
```diff
@@ -1,60 +1,89 @@
 ﻿# Angular for XMCloud
 
-> Sitecore JSS Angular App for XM Cloud. For the current release this feature is experimental.
+> Sitecore JSS Angular App for XM Cloud. For the current release this feature is in public beta.
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: "For the current release this feature is in public beta" > "This feature is currently in public beta."

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 5]
```diff
@@ -1,60 +1,89 @@
...
+> If you encounter any issues or bugs, please submit a new issue in the [Jss GitHub Repo](https://github.com/Sitecore/jss)
 
-[Documentation](<TODO>)
+[Documentation](https://doc.sitecore.com/xmc/en/developers/jss/latest/jss-xmc/introducing-sitecore-javascript-rendering-sdk.html)
 
-This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration. The supported key features are as follows:
+This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration.
+
+The Angular XM Cloud integration consists of two parts:
+- `XM Cloud Angular`: simplifies connecting the application to XM Cloud and configuring the integration of multiple composable Sitecore products, provides out of the box helper components.
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: "provides out of the box helper components." > "and provides out-of-the-box helper components."

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 6]
```diff
@@ -1,60 +1,89 @@
...
+- `XM Cloud Proxy`: Adds integration with XM Cloud for the JSS SPA applications and enables editing, personalization and A/B component testing support, see [XM Cloud Proxy](../node-xmcloud-proxy/)
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: ", see [XM Cloud Proxy](../node-xmcloud-proxy/)" > ". See [XM Cloud Proxy](../node-xmcloud-proxy/) for more information."

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 7]
```diff
@@ -1,60 +1,89 @@
...
+
+The supported key features are as follows:
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: "The following key features are supported:"

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 8]
```diff
@@ -1,60 +1,89 @@
...
 - `XM Cloud Pages editing integration`: full integration with Pages - the dynamic visual page editor of XM Cloud.
 
-- `XM Cloud proxy personalization` with embedded personalization and A/B Component Test support.
+- `XM Cloud proxy personalization` with embedded personalization and component A/B/n testing.
 
-- `Forms support`: provides the capability to consume and post Sitecore Forms from JSS apps. Sitecore Forms is a form-authoring framework that enables marketers to author their own forms, collect data, and analyze form performance.
+- `Forms support`: consume and post XM Cloud Forms in JSS apps. Using Forms, marketers can create forms, collect data, and analyze form performance.
 
-This SPA is tailored to enhance development workflows and enable full utilization of XM Cloud’s capabilities, providing a seamless and efficient foundation for developers.
+- `Internationalization` support
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: Add full-stop.

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 9]
```diff
@@ -1,60 +1,89 @@
...
 
-## Components and Supporting Applications
+> The following features and integrations are not supported by Angular and Proxy apps for XM Cloud:
+> - Multisite
+> - XM Cloud Components application
+> - BYOC components
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md:29]: Confirm with Anastasia whether we need to say this in addition to "The XM Cloud Components application", as Liz stated this is the same thing.

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 10]
```diff
@@ -1,60 +1,89 @@
...
-- `Forms support`: provides the capability to consume and post Sitecore Forms from JSS apps. Sitecore Forms is a form-authoring framework that enables marketers to author their own forms, collect data, and analyze form performance.
+- `Forms support`: consume and post XM Cloud Forms in JSS apps. Using Forms, marketers can create forms, collect data, and analyze form performance.
 
-This SPA is tailored to enhance development workflows and enable full utilization of XM Cloud’s capabilities, providing a seamless and efficient foundation for developers.
+- `Internationalization` support
 
-## Components and Supporting Applications
+> The following features and integrations are not supported by Angular and Proxy apps for XM Cloud:
+> - Multisite
+> - XM Cloud Components application
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: "The XM Cloud Components application"

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 11]
```diff
@@ -1,60 +1,89 @@
...
-| `SITECORE_SITE_NAME`                              | The name of your site. This variable overrides the config.appName defined in the package.json file. You can find this value in the Sites dashboard by opening the actions menu for a site and navigating to Settings > Developer settings. Default value `<appName>`,
-`                  |
-| `DEFAULT_LANGUAGE`                              | The default language of your app. Default value `en`                  |
-| `DEBUG`                  | Optional. Debug level for the proxy. Set the DEBUG environment variable to 'sitecore-jss:*,proxy*,http-proxy-middleware*' to see all logs. Refer to the [official docs](https://doc.sitecore.com/xp/en/developers/hd/latest/sitecore-headless-development/debug-logging-in-jss-apps.html#namespaces) for all the available namespaces.
+### Production
+
+To build and run in production mode you need to have your angular app side by side with the [Node XM Cloud proxy](../node-xmcloud-proxy/).
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: "Angular"

What do we mean by "side by side"?

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 12]
```diff
@@ -1,60 +1,89 @@
...
 
-When building and running your app in connected mode the proxy application is not needed. However, please note that certain features, such as personalization, server-side rendering, and editing, will not be available.
+For additional information on how to set up and run a SPA app in production mode against an XM Cloud instance you can check the [spa-starters monorepo](https://github.com/sitecorelabs/xmcloud-foundation-head/tree/main/headapps/spa-starter) in the XM Cloud Foundation Starter Kit.
 
-1. Run `npm install`
-2. Run `npm run build`
-3. Run `npm run start:connected`
+## Environment Variables
+
+The following environment variables can be set to configure the angular app. You can use the `.env` file located in the root of the app or set these directly in the environment (for example, in containers).
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: "can be used to configure"

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 13]
```diff
@@ -1,60 +1,89 @@
...
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `PROXY_HOST`                        | Your XM Cloud Proxy hostname is needed to build the app and execute the client-side requests against the proxy server. Default value `http://localhost:3000`                                                                                                                  |
+| `PROXY_BUILD_PATH`                              | Your XM Cloud Proxy server path is needed to build the app. The build output will be copied to the XM Cloud Proxy application path. Default value `<xmcloud_proxy_path>\dist`.
+| `SITECORE_EDGE_CONTEXT_ID`                              | The Context ID, which covers many system configurations, required for connecting to the XM Cloud back end. This is an XM Cloud system environment variable. When the application runs on the XM Cloud editing host, this value is always set to the preview Context ID.                   |
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: "...is required for connecting to the XM Cloud back end."

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md [hunk 14]
```diff
@@ -1,60 +1,89 @@
...
+| `SITECORE_API_KEY`                              | The API key for GRAPH_QL_ENDPOINT authentication. It should be used in combination with SITECORE_API_HOST for local development when connecting to a local XM Cloud instance
+| `SITECORE_API_HOST`                              | The API hostname, needed to build the application. This should be used in combination with SITECORE_API_KEY for local development. For example, https://<xmc_cm_host>.sitecorecloud.io.                   |
+| `GRAPH_QL_ENDPOINT`                              | Your GraphQL Edge endpoint. This is typically optional. By default, the endpoint is calculated using the resolved Sitecore API hostname + the `graphQLEndpointPath` defined in your `package.json`. For a preview endpoint (a CM instance on XM Cloud or a local one), the value is <xmc_cm_host>/sitecore/api/graph/edge.  |
+| `SITECORE_SITE_NAME`                              | The name of your site. This variable overrides the config.appName defined in the package.json file. You can find this value in the Sites dashboard by opening the actions menu for a site and navigating to Settings > Developer settings. The default value is the name of your app. ,
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: Check , and ` at end of this line.

File: packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md [hunk 15]
```diff
@@ -1,32 +1,41 @@
...
+> Sitecore XM Cloud Proxy is in public beta.
+> If you encounter any issues or bugs, please submit a new issue in the [Jss GitHub Repo](https://github.com/Sitecore/jss)
 
-[Documentation](TODO)
+[Documentation]((https://doc.sitecore.com/xmc/en/developers/jss/latest/jss-xmc/introducing-sitecore-javascript-rendering-sdk.html)) 
 
-This proxy will serve as the backbone for supporting various SPA frameworks by handling server-side rendering (SSR), data queries, and middleware functionalities.
+This Node.js-based proxy app is the backbone for enabling seamless integration between XM Cloud and various SPA frameworks like React, Angular, or Vue laying the groundwork for future JSS starter kits built for other front-end JavaScript frameworks. It acts as a middleware layer to handle critical functionalities such as server-side rendering (SSR), enabling editing and personalization, A/B component testing, and integrating Sitecore Forms. By serving as the rendering host, it ensures a smooth connection between Sitecore XM Cloud services and your front-end applications, making it easier to build dynamic, personalized, and localized experiences for users.
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md]: "A/B/n"

File: packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md [hunk 16]
```diff
@@ -1,32 +1,41 @@
...
+This is a sample setup showing one of how you can configure XM Cloud rendering server on top of Node.js and Express.js
 
 ## Features Supported
 
 - `Context ID`: The Context ID environment variable simplifies setting up and configuring XM Cloud solutions. It's a unified identifier that maps to all your configured resources, such as content, sites, files, forms, and integration settings.
 
 - `XM Cloud Pages editing integration`: full integration with Pages - the dynamic visual page editor of XM Cloud.
 
-- `XM Cloud proxy personalization` with embedded personalization and A/B Component Test support.
+- `XM Cloud proxy personalization` with embedded personalization and Component A/B Testing support.
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md]: "component A/B/n testing"

File: packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md [hunk 17]
```diff
@@ -1,32 +1,41 @@
...
 - `Forms support`: provides the capability to consume and post Sitecore Forms from JSS apps. Sitecore Forms is a form-authoring framework that enables marketers to author their own forms, collect data, and analyze form performance.
 
-## Configuration Setup
+- `Internationalization` support
 
-The config.ts file in this proxy app handles essential configurations for server-side rendering, data queries, and middleware functionalities. Here are the main configurations defined:
+> The following features and integrations are not supported by Node XM Cloud Proxy:
+> - Multisite
+> - XM CLoud Components application
+> - BYOC components
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md:27]: As elsewhere, please confirm with Anastasia whether we need to say BYOC here as well as the components app.

File: packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md [hunk 18]
```diff
@@ -1,32 +1,41 @@
...
 
 - `Forms support`: provides the capability to consume and post Sitecore Forms from JSS apps. Sitecore Forms is a form-authoring framework that enables marketers to author their own forms, collect data, and analyze form performance.
 
-## Configuration Setup
+- `Internationalization` support
 
-The config.ts file in this proxy app handles essential configurations for server-side rendering, data queries, and middleware functionalities. Here are the main configurations defined:
+> The following features and integrations are not supported by Node XM Cloud Proxy:
+> - Multisite
+> - XM CLoud Components application
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md]: "The XM Cloud Components application"

File: packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md [hunk 19]
```diff
@@ -49,21 +58,25 @@ This configuration is designed to be flexible and secure, with dynamic settings
 
 ### Environment Variables
 
-The following environment variables can be set to configure the Proxy sample instead of modifying `config.js`. You can use the `.env` file located in the root of the app or set these directly in the environment (for example, in containers).
+The following environment variables can be set to configure the Node XM Cloud Proxy app instead of modifying `config.js`. You can use the `.env` file located in the root of the app or set these directly in the environment (for example, in containers).
```
Comment (mattcandoit) [inline packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md]: "can be used to configure"

**Review by mattcandoit — CHANGES_REQUESTED** [review]
Most comments relate to Angular. I added some to the Node proxy readme too, but these are mostly addressed in this Word doc: https://sitecore1-my.sharepoint.com/:w:/g/personal/matp_sitecore_net/EYtEb9QYkGFLtNAW5AcyTU8BVp8ciAuaKI4g9Qxz4eY3Tg?e=s0Koma

File: packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md (hunk 3 above)
Comment (anasplat) [reply packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md]: # Angular for XMCloud>Angular for XM Cloud

**Pull request merged**